aggregator/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Which part of the test code requires this?

Reply:
we need it to mock static method for test `DailyEventAggregationJobTest.validateScheduleExecutionTimeAdjustment`